README.md

```diff
@@ -22,8 +22,9 @@ For more information, please read [Building a UI Component in 2017 and Beyond](h
 * [`ebay-breadcrumb`](https://github.com/eBay/ebayui-core/tree/master/src/components/ebay-breadcrumb)
 * [`ebay-button`](https://github.com/eBay/ebayui-core/tree/master/src/components/ebay-button)
 * [`ebay-menu`](https://github.com/eBay/ebayui-core/tree/master/src/components/ebay-menu)
-* [`ebay-select`](https://github.com/eBay/ebayui-core/tree/master/src/components/ebay-select)
 * [`ebay-notice`](https://github.com/eBay/ebayui-core/tree/master/src/components/ebay-notice)
+* [`ebay-pagination`](https://github.com/eBay/ebayui-core/tree/master/src/components/ebay-pagination)
+* [`ebay-select`](https://github.com/eBay/ebayui-core/tree/master/src/components/ebay-select)
```

Comment:
@ianmcburnie and I love you for making this alphabetical.

Reply:
I learned from the best 😉